estimatey-client/src/appPaths.tsx

```diff
@@ -0,0 +1,19 @@
+import { useParams } from "react-router-dom";
+
+const appPaths = {
+    home: "/",
+    projectFeatures: (projectId: string | number) => `/projects/${projectId}/features`,
+}
+
+const createUseId = (paramName: string) => (): number => {
```

Comment:
Nit pick but seems like a weird name to me, could it be something like `useParamAsNumber` instead?